tests/validation/tests/v3_api/test_node_template.py

```diff
@@ -0,0 +1,149 @@
+import os
+import pytest
+from rancher import ApiError
+from .common import *  # NOQA
+from .test_rbac import create_user
+
+DO_ACCESSKEY = os.environ.get('DO_ACCESSKEY', "None")
+
+user_token = {"stduser_with_createrketemplate_role": {"user": None,
+                                                      "token": None},
+              "standard_user": {"user": None, "token": None}}
+
+@pytest.fixture(scope='module', autouse="True")
+def setup(request):
+
+    client = get_admin_client()
+
+    # create users
+    user_token["stduser_with_createrketemplate_role"]["user"], \
+        user_token["stduser_with_createrketemplate_role"]["token"] = \
+        create_user(client)
+    user_token["standard_user"]["user"], \
+        user_token["standard_user"]["token"] = create_user(client)
+
+    stduser_with_createrketemplate_role_id = \
+        user_token["stduser_with_createrketemplate_role"]["user"].id
+
+    # Add clustertemplates-create global role binding to the standard user
```

Comment:
why do we need `clustertemplates` here?

Reply:
Mistakenly taken from wrong code. Addressed in changes.